easybuild/easyconfigs/d/DIAL/DIAL-2011.06.06-foss-2016a.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'ConfigureMake'
+
+name = 'DIAL'
+version = '2011.06.06'
+
+homepage = 'http://www.bx.psu.edu/miller_lab/'
+description = """DIAL (De novo Identification of Alleles) is a collection of programs
+ to automate the discovery of alleles for a species where we lack a reference sequence. 
+ The SNPs/alleles are specifically selected for a low error rate in genotyping assays.  
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://www.bx.psu.edu/miller_lab/dist/']
+
+patches = ['DIAL-Makefile.in.patch']
```

Comment:
@RvDijk please rename to include the DIAL version: `DIAL-%(version)s_Makefile.in.patch`